### PR TITLE
Add SECURITY.md to the repository

### PR DESCRIPTION
**Pull request by nbougalis**
Document the security policy for `rippled`, outline the Ripple-sponsored bug-bounty program and provide contact information for researchers who wish to responsibly disclose issues or security vulnerabilities that they may identify.

For more about security policies on GitHub, please visit [Adding a security policy to your repository](https://help.github.com/en/github/managing-security-vulnerabilities/adding-a-security-policy-to-your-repository)

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+- The description of the vulnerability;
+- The attack scenario (if any);
+- The steps to reproduce the vulnerability;
+- Any other relevant details or artifacts, including code, scripts or patches.
+
+In your mail, please describe of the issue or the potential threat; if possible, please include a "repro" (code that can reproduce the issue) or describe the best way to reproduce and replicate the issue. Please make your report as extensive as possible.
+
+For more information on responsible disclosure, please read this [Wikipedia article](https://en.wikipedia.org/wiki/Responsible_disclosure).
+
+## Report Handing Process
```
Comment (MarkusTeufelberger) [inline SECURITY.md:44]: Did you mean

```suggestion
## Report Handling Process
```

?

**Comment by MarkusTeufelberger** [comment]
Including the logo is nice and all, but it is under a different license than the rest of the code and documentation of this repository. It might be nice not to put in (even more) copyrighted material in here, but in the end that's up to you and your lawyers.

File: SECURITY.md (hunk 1 above)
Comment (nbougalis) [reply SECURITY.md]: Yep, thanks. Acccepted the suggestion.

**Comment by nbougalis** [comment]
That's a good point, re: logo license. I think that in the context it's used, it doesn't _really_ matter, but I am not a lawyer and I didn't sleep at a Holiday Inn Express last night, so...

Anyways, I don't think the logo necessarily adds to much in terms of value here and can just as easily go away. I'll push a FOLD commit that does that.

Thanks for the review @MarkusTeufelberger.

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
+### Operating an XRP Ledger server securely
+
+For more details on operating an XRP Ledger server securely, please visit https://xrpl.org/manage-the-rippled-server.html.
+
+
+# Security Policy
+
+## Supported Versions
+
+Software constantly evolves. In order to focus resources, we only generally only accept vulnerability reports that affect recent and current versions of the software. We always accept reports for issues present in the **master**, **release** or **develop** branches, and with proposed, [open pull requests](https://github.com/ripple/rippled/pulls).
```
Comment (carlhua) [inline SECURITY.md:10]: Consider removing "we only accept vulnerability reports for recent." An older version can still have security-related issues. Per Ed, maybe we can do something like "we will only consider vulnerability reports for old version if the vulnerability still exists in the recent and current versions.". The idea here is not to discourage people from submitting issues.

**Review by carlhua — APPROVED** [review]
LGTM

**Review by carlhua — APPROVED** [review]
Actually I have one more comment - in order to make the release process more streamlined, can we consider a better description for the PR. Even if it something as trivial as this one. This will help the person that makes the release life better

File: SECURITY.md (hunk 2 above)
Comment (nbougalis) [reply SECURITY.md:10]: Soliciting the reporting of vulnerabilities that are not present in a recent version of the software (i.e. are no longer present in the source code) will only serve to bottleneck the triage and evaluation process, with little to no actual benefit.

By way of example, assume a bug against **1.2.4** is reported _today_ which was fixed in **1.4.0**. The **1.5.0** release is out and **1.6.0** is in beta. The only actionable item from a report would be "please upgrade to **1.5.0** which does not contain this bug."

**Pull request closed without merging**